### PR TITLE
Analytics: send local node room state/info

### PR DESCRIPTION
**Pull request by shishirng**

### CODE REVIEW

File: pkg/telemetry/analyticsservice.go [hunk 1]
```diff
@@ -28,14 +31,17 @@ import (
...
-	nodeID       string
+	analyticsKey   string
+	nodeID         string
+	sequenceNumber atomic.Uint64
 
-	events livekit.AnalyticsRecorderService_IngestEventsClient
-	stats  livekit.AnalyticsRecorderService_IngestStatsClient
+	events    livekit.AnalyticsRecorderService_IngestEventsClient
+	stats     livekit.AnalyticsRecorderService_IngestStatsClient
+	nodeRooms livekit.AnalyticsRecorderService_IngestNodeRoomStatesClient
```
Comment (davidzhao) [inline pkg/telemetry/analyticsservice.go:44]: just curious, why do these need different clients?

Comment (davidzhao) [reply pkg/telemetry/analyticsservice.go:44]: and I don't see them initialized.. are there API to set them elsewhere?

**Review by davidzhao — APPROVED** [review]
lg, just a couple of questions

File: pkg/telemetry/analyticsservice.go (hunk 1 above)
Comment (shishirng) [reply pkg/telemetry/analyticsservice.go:44]: these are streaming apis, so each get their own client

**Review by davidzhao — APPROVED** [review]

**Pull request merged**